From 73403e8de2c5bc6d54f89ae2f1c9aa12b92ecb7a Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Mon, 7 Jul 2014 08:27:43 -0400
Subject: [PATCH] resolved: Move symlink creation from tmpfiles to daemon
 runtime

At least Fedora right now doesn't by default use resolved; the service
is disabled by default in the 90-default.preset file.

The change to unconditionally create the resolv.conf symlink broke
Anaconda and related tools (lorax) which expect it to be a regular
file.  In particular, Anaconda expects to be able to persist
networking state from the installation environment to the target
system.
---
 tmpfiles.d/etc.conf            | 1 -
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/tmpfiles.d/etc.conf b/tmpfiles.d/etc.conf
index b23272cb27..125d6e0a17 100644
--- a/tmpfiles.d/etc.conf
+++ b/tmpfiles.d/etc.conf
@@ -10,6 +10,5 @@
 L /etc/os-release - - - - ../usr/lib/os-release
 L /etc/localtime - - - - ../usr/share/zoneinfo/UTC
 L+ /etc/mtab - - - - ../proc/self/mounts
-L /etc/resolv.conf - - - - ../run/systemd/resolve/resolv.conf
 C /etc/nsswitch.conf - - - -
 C /etc/pam.d - - - -
-- 
1.9.3

